canonical-web-and-design/cn.ubuntu.com
# Change phone label to mobile phone

## Done

- Change 'phone' label to 'mobile phone'

## QA

- download this branch
- go to http://0.0.0.0:8010/contact
- see that phone is now mobile phone (use google translate)

## Issue / Card

Fixes #382
